src/elements/content-preview/__tests__/ContentPreview.test.js

```diff
@@ -1146,12 +1147,14 @@ describe('elements/content-preview/ContentPreview', () => {
                     },
                 };
 
+                instance.getViewer = jest.fn().mockReturnValue({ emit });
```

Comment:
I've been tending to prefer `jest.spyOn` lately since it leaves the underlying method in place. Should we test the case where the viewer is undefined?